.changeset/ten-goats-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/docs": major
```

Comment:
This can be considered a `minor` NOT `major`

Reply:
Thanks for the review this is my first time,I've made those changes

Reply:
That's totally okay, buddy.

Thank you for taking the time to contribute 💖